Source/compositorbuffer/include/compositorbuffer/CompositorBufferType.h

```diff
@@ -72,6 +68,13 @@ namespace Compositor {
             SharedStorage& operator=(const SharedStorage&) = delete;
 
             SharedStorage()
+                : _width(0)
+                , _height(0)
+                , _format(0)
+                , _modifier(0)
+                , _type(Exchange::ICompositionBuffer::TYPE_INVALID)
+                , _dirty()
+                , _copyOfDirty()
```

Comment:
Better to explicitly set it  to a value for clarity (at the moment it would be false I assume)